src/test/java/picard/util/FifoBufferTest.java

```diff
@@ -57,8 +57,8 @@ public void test(final double megabytes) throws IOException {
         }
 
         // Run the input file through the FifoBuffer and back into another file
-        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), null);
-        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), null);
+        final Md5CalculatingInputStream in   = new Md5CalculatingInputStream(new FileInputStream(inputFile), (File)null);
+        final Md5CalculatingOutputStream out = new Md5CalculatingOutputStream(new FileOutputStream("/dev/null"), (File)null);
```

Comment:
`FifoBufferTest.testFifoBuffer` should throw a NPE since this `Md5CalculatingOutputStream` constructor uses the statement `digestFile.toPath()` and `digestFile` in null. But, there are no errors in the test results. Did travis not run the tests? Since you fixed the NPE in https://github.com/samtools/htsjdk/pull/881/files you can either grab the latest `htsjdk SNAPSHOT` or cast the `null` to `Path`.